libheif/heif_colorconversion.cc

```diff
@@ -306,7 +306,7 @@ static inline uint8_t clip_int_u8(int x)
 
 static inline uint16_t clip_f_u16(float fx, int32_t maxi)
 {
-  long x = lroundf(fx);
+  long x = (long int) (fx + 0.5);
```

Comment:
Should be 0.5f to avoid any implicit cast to double (and same below)

Reply:
Fixed